.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "devcontainer-fluent-bit",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
```

Comment:
Why this particular base? Is it specifically for dev containers or can we use the standard debian base? It could tie into the builder stage then maybe?

Reply:
It is my up & ready config :) we are able to use debian base. I will try to use fluent/fluent-bit:latest-debug.

Reply:
I updated PR. it works :rocket: